Index: roundcube-amacube/localization/de_DE.inc
===================================================================
--- roundcube-amacube.orig/localization/de_DE.inc
+++ roundcube-amacube/localization/de_DE.inc
@@ -11,7 +11,7 @@ See the COPYING file for a full license
 */
 
 $labels = array();
-$labels['amavissettings_pagetitle'] = 'Amavis Einstellungen';
+
 $labels['policy_default_message'] = 'Bisher wurden keine Benutzerspezifischen Einstellungen gefunden, verwende Voreinstellungen.';
 $labels['section activate'] = 'Aktivierung';
 $labels['spam_check'] = 'Aktiviere Spamprüfung';
@@ -30,7 +30,7 @@ $labels['virus_quarantine_active_info']
 $labels['banned_quarantine'] = 'Quarantäne von gefährlichen Inhalten';
 $labels['banned_quarantine_active'] = 'Quarantäne von gefährlichen Inhalten';
 $labels['banned_quarantine_active_info'] = 'Klick hier um Emails mit gefährlichen Inhalten in die Quarantäne zu nehmen';
-$labels['section adjust levels'] = 'Festlegen der Spamwerstufen';
+$labels['section adjust levels'] = 'Festlegen der Spamwehrstufen';
 $labels['spam_tag2_level'] = 'Spamwert für Spamverdacht';
 $labels['spam_tag2_level_info'] = 'Emails mit einem Spamwert in dieser Höhe oder mehr könnte spam sein, wird aber noch zugestellt. Ein X-Spam-Header mit dem Wert Yes wird hinzugefügt.';
 $labels['spam_tag2_level_error'] = 'Der Wert für Spamverdacht muss eine ganze Zahl zwischen -20 und 20 sein.';
@@ -40,9 +40,9 @@ $labels['spam_kill_level_error'] = 'Der
 $labels['verification_error'] = 'Die Amaviseinstellungen enthielten einen Fehler und konnten nicht gespeichert werden.';
 $labels['write_error'] = 'Es kam zu einem Fehler beim Abspeichern der Amaviseinstellungen';
 $labels['successfully saved'] = 'Die Amaviseinstellungen für diesen Benutzer wurden erfolgreich abgespeichert.';
-$labels['quarantine heading'] = 'Quarantäne-Emails';
-$labels['quarantine_no_result'] = 'Es konnten keine Emails in der Quarantäne gefunden werden';
-$labels['quarantine_pagetitle'] = 'Quarantäne-Emails';
+$labels['quarantine heading'] = 'Quarantäne E-Mails';
+$labels['quarantine_no_result'] = 'Es konnten keine E-Mails in der Quarantäne gefunden werden';
+$labels['quarantine_pagetitle'] = 'Quarantäne E-Mails';
 $labels['intersection_error'] = 'Fehler: Es kann eine E-Mail nicht gleichzeitig gelöscht und aus der Quarantäne freigegeben werden!';
 $labels['content_decode_V'] = 'Virus';
 $labels['content_decode_B'] = 'Verbotener Inhalt';
@@ -55,5 +55,46 @@ $labels['content_decode_O'] = 'Größenb
 $labels['content_decode_T'] = 'MTA Fehler';
 $labels['content_decode_C'] = 'Sauber';
 
+$labels['filter_settings_pagetitle'] = 'Spam-Filter';
+$labels['section_checks'] = 'Prüfungen';
+$labels['section_quarantine'] = 'Quarantäne';
+$labels['section_delivery'] = 'Zustellung';
+$labels['section_levels'] = 'Levels';
+$labels['spam_quarantine_cutoff_level'] = 'Spamwert für zulöschende Spam';
+$labels['process'] = 'Verarbeiten';
+$labels['quarantine'] = 'Quarantäne';
+$labels['spam_delivery'] = 'E-Mails mit Spam';
+$labels['virus_delivery'] = 'E-Mails mit Viren';
+$labels['banned_delivery'] = 'E-Mails mit verbotenen Dateien';
+$labels['bad_header_delivery'] = 'E-Mails mit ungültigen Kopfzeilen';
+$labels['deliver'] = 'Zustellen';
+$labels['discard'] = 'Löschen';
+$labels['process'] = 'Verarbeiten';
+$labels['email_filter_disabled'] = 'E-Mail Filter abgeschaltet';
+$labels['email_filter_enabled'] = 'E-Mail Filter eingeschaltet';
+$labels['process_quarantine'] = 'Ausgewählte Nachrichten werden freigegeben oder gelöscht';
+$labels['release'] = 'Freigeben';
+$labels['delete'] = 'Löschen';
+$labels['subject'] = 'Betreff';
+$labels['received'] = 'Erhalten';
+$labels['mailtype'] = 'Mail typ';
+$labels['spamlevel'] = 'Spamwert';
+$labels['messages'] = 'Nachrichten';
+$labels['levels'] = 'Levels';
+$labels['status'] = 'Status';
+$labels['checks'] = 'Prüfungen';
+$labels['to'] = 'an';
+$labels['of'] = 'von';
+$labels['success_release'] = 'Ausgewählte Nachrichten wurden freigegeben.';
+$labels['success_delete'] = 'Ausgewählte Nachrichten wurden gelöscht.';
+
+$labels['db_conect_error'] = 'Datenbankverbindungsfehler.';
+$labels['db_query_error'] = 'Datenbankabfragefehler.';
+$labels['db_policy_error'] = 'Datenbankrichtlinienfehler.';
+$labels['spam_quarantine_cutoff_level_error'] = 'Der Wert für zulöschende Spammails muss eine ganze Zahl zwischen 20 und 1001 sein.';
+
+$labels['amavissettings_pagetitle'] = 'Amavis Einstellungen';
+
+
 
 ?>
